qa/rpc-tests/miningtest.py

```diff
@@ -173,7 +174,8 @@ def Test():
     t = MyTest()
```

Comment:
not: while at it why not using something else rather then `MyTest`? what about `MiningTest`?

Reply:
done